CMakeLists.txt

```diff
@@ -171,39 +173,49 @@ function(add_valgrind_test NAME_WITH_PREFIX BINARY WORKING_DIRECTORY)
     endif()
 endfunction()
 
-get_target_property(QMAKE Qt::qmake IMPORTED_LOCATION)
-set(CARGO_ENV "QMAKE=set:${QMAKE}")
-set(RUNTIME_ENV "")
```

Comment:
why are these still not set all the time?

This shouldn't "break" wasm builds by setting the QMAKE path ? As we are still seeing it use the wrong one anyway aren't we ?